PatientWebApplication/HealthClinic.DAL/Utility/UtilityMethods.cs

```diff
@@ -23,5 +24,16 @@ public static Boolean TryParseDateInCorrectFormat(String dateString)
             var date = new DateTime();
             return DateTime.TryParseExact(dateString, "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out date);
         }
+
+        public static Boolean CheckForSpecialty(DoctorUser doctor, string specialty)
+        {
+            return doctor.speciality.Equals(specialty);
+        }
+
+        public static Boolean TryParseTimeSpanInCorrectFormat(String timeSpan)
+        {
+            var time = new TimeSpan();
+            return TimeSpan.TryParseExact(timeSpan, "hh:\\mm\\:ss", CultureInfo.InvariantCulture, out time);
```

Comment:
Write in the form:
return TimeSpan.TryParseExact(timeSpan, "hh:\\mm\\:ss", CultureInfo.InvariantCulture, out new TimeSpan());